common/djangoapps/student/views/dashboard.py

```diff
@@ -528,7 +528,10 @@ def student_dashboard(request):  # lint-amnesty, pylint: disable=too-many-statem
     """
     user = request.user
     if not UserProfile.objects.filter(user=user).exists():
-        return redirect(settings.ACCOUNT_MICROFRONTEND_URL)
+        return redirect(configuration_helpers.get_value(
```

Comment:
why get this value multiple times? All 3 calls you do that use this  value are in the same function. Just get the value once and stick it in a variable.